lib/cisco_node_utils/bgp_af.rb

```diff
@@ -650,5 +723,60 @@ def route_target_delta(should, is, prop)
         end
       end
     end
+
+    #
+    # Suppress Inactive (Getter/Setter/Default)
+    #
+    def suppress_inactive
+      state = config_get('bgp_af', 'suppress_inactive', @get_args)
+      state ? true : false
+    end
+
+    def suppress_inactive=(state)
+      state = (state ? '' : 'no')
+      set_args_keys(state: state)
+      config_set('bgp_af', 'suppress_inactive', @set_args)
+    end
+
+    def default_suppress_inactive
+      config_get_default('bgp_af', 'suppress_inactive')
+    end
+
+    #
+    # Table Map (Getter/Setter/Default)
+    #
+
+    def table_map
+      config_get('bgp_af', 'table_map', @get_args)
+    end
+
+    def table_map_filter
+      config_get('bgp_af', 'table_map_filter', @get_args)
+    end
+
+    def table_map_set(map, filter=false)
+      if filter
+        feature = 'table_map_filter'
```

Comment:
Nit: I wouldn't use `feature` here since that has a different connotation. Consider using `attr` instead.